common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -324,6 +324,17 @@
           "The change log writer version cannot be <version>."
         ]
       },
+      "KEY_ROW_FORMAT_VALIDATION_FAILURE" : {
+        "message" : [
```

Comment:
Let's avoid duplicating this message here. We already have another error class with this same message, so instead of duplicating, we can pass the message from that exception instance into this exception instance. I have added another comment in the code where you do the exception creation to give more context